test-stones-even-closer-together.sh

```diff
@@ -2,11 +2,19 @@
 
 # conveyor-speed=58mm/s
 
+start_hardware () {
+    echo "conveyor=running"
+    echo "compressor=start"
+}
+
+stop_hardware () {
+    echo "conveyor=stopped"
+    echo "compressor=stop"
+}
 run_blue_stone () { 
     echo "lightbarrier1=on"
-    sleep .45
+    sleep .1
```

Comment:
Why was this time shortened?
The conveyor runs at a constant speed so this time should be the same for all three scripts